[EDU-1630] - Spaces Member Location example (#2291)
ably/docs: examples/spaces-member-location/page.md

# Member location for applications

> Track the locations of other users in an application.

Reviewer: ```suggestion
Track the location of other users within an application.
```

> Track and display the location of users within an application, such as a form field they have selected, the cell they're currently editing in a spreadsheet, or the slide they're viewing on a slide deck.

Reviewer: ```suggestion
Track and display the location of users within an application, such as which form field they have selected, the cell they're currently editing in a spreadsheet, or the slide they're viewing on a slide deck.
```

> Member location are implemented using [Ably Spaces](https://ably.com/docs/products/spaces). The Spaces SDK contains a set of purpose-built APIs that abstract away the complexities involved in managing the locations of participants focus in a collaborative application. It is built on top of Ably's core platform, and so it provides the same performance and scaling guarantees.

Reviewer: ```suggestion
Member locations are implemented using [Ably Spaces](https://ably.com/docs/products/spaces). The Spaces SDK contains a set of purpose-built APIs that abstract away the complexities involved in managing the locations of participants focus in a collaborative application. It is built on top of Ably's core platform, and so it provides the same performance and scaling guarantees.
```

## Resources

// React brief (Only visible if viewing the React example)

Use the following components to add member location into an application:

* [`SpacesProvider`](https://ably.com/docs/spaces/react#spaces-provider): initializes and manages a shared space client instance, passing it down through React context to enable real-time spaces functionality across the application.

> * [`SpaceProvider`](https://ably.com/docs/spaces/react#spaces-provider): manages the state and functionality of a specific space, providing access to real-time interactions within that space via React context.

Reviewer: real-time --> realtime
